tox.ini

```diff
@@ -1,23 +1,20 @@
 [tox]
-envlist = py{35,38}-pylint{24}, coverage, pylint
+envlist = py38-pylint{24}, coverage, pylint
 
 [testenv]
-deps =
-    -r{toxinidir}/requirements/test.txt
-
-commands =
-    # edx_lint itself pins the pylint version, so we have to re-install pylint
-    # to test on different versions.
-    pylint24: pip install -q pylint>=2.4,<2.5
-    pylint --version
-    coverage run -p -m pytest {posargs:}
+deps = 
+	-r{toxinidir}/requirements/test.txt
```

Comment:
This is putting tabs into the file.  It shouldn't.

Reply:
I've made a pull request for repo-tools to fix this: https://github.com/edx/repo-tools/pull/157